requirements-ci.txt

```diff
@@ -1,10 +1,10 @@
-numpy
-scipy<1.13.0
+numpy~=2.0.0
```

Comment:
We likely shouldn't pin the version here.

Reply:
I thought we wanted to keep it at some 2.0.x version to have it compatible with Autograd